odl/__init__.py

```diff
@@ -41,6 +42,8 @@
 from . import phantom
 from . import deform
 from . import ufunc_ops
+from . import datasets
```

Comment:
Actually, when I think about it, phantoms are a totally different thing (e.g. space-wise) than data, so it's only logical to keep them separate. So even humanoid phantoms are phantoms and go into `phantoms`, whereas `datasets` can also be simulated data.

So I'd say it's fine like this.